CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## Next
+- Fix `zls` not restarting after having been updated on macOS (@ngrilly)
```

Comment:
0.3.2 hasn't been released yet so this would go under it.

Reply:
Thanks! I moved it under 0.3.2.